lib/journald/reader.go

```diff
@@ -81,6 +81,13 @@ func (r *reader) getMessage() (msg lib.Message, ok bool, err error) {
 		return
 	}
 
+	if task, err := r.GetDataValue("CONTAINER_TASK"); err == nil && len(task) != 0 {
+		// If we have task information, use that as the stream name instead
+		msg.Stream = arnToUUID(task)
+	}
+
+	msg.Stream = sanitizeStreamName(msg.Stream)
```

Comment:
Without this change, you can still break `ecs-logs` if your container name has ":" or "*" in it, doesn't really seem unique to task.